NEXT_CHANGELOG.md

```diff
@@ -43,6 +43,35 @@ so it should not change depending on the variables or HTTP headers.
 
 By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1439
 
+### Change `PluggableRouterServiceBuilder` methods ([PR #1437](https://github.com/apollographql/router/pull/1437))
+
+`with_naive_introspection` and `with_defer_support` where two parameter-less methods
+of this builder that enabled boolean configuration flags.
+They have been removed and replaced by `with_configuration` 
+which takes `Arc<apollo_router::Configuration>`.
+A `Configuration` value can be created from various formats by deserializing with `serde`.
+The removed methods correspond to `server.introspection` and `server.experimental_defer_support`
+configuration keys respectively.
+
+By [@SimonSapin](https://github.com/SimonSapin) in https://github.com/apollographql/router/pull/1437
+
+### The `SchemaKind` enum had its `Instance` variant replaced with a `String` ([PR #1437](https://github.com/apollographql/router/pull/1437))
+
+As the name suggests, the new variant contains a `String` instead of `Box<apollo_router::Schema>`,
+so you no longer need to parse the schema before giving it to the router.
+
+By [@SimonSapin](https://github.com/SimonSapin) in https://github.com/apollographql/router/pull/1437
+
+### `Schema` no longer implements `FromStr` ([PR #1437](https://github.com/apollographql/router/pull/1437))
+
+This means that `str.parse::<apollo_router::Schema>()` is no longer available.
+If you still need a parsed `Schema` (see above),
+use `apollo_router::Schema(str, &configuration)` instead.
+To use the default `apollo_router::Configuration` 
+you can call `apollo_router::Schema(str, &Default::default())`.
+
+By [@SimonSapin](https://github.com/SimonSapin) in https://github.com/apollographql/router/pull/1437
```

Comment:
I like these changes, but you probably want more opinions than just mine on these changes...